src/core/server/models/comment/comment.ts

```diff
@@ -472,6 +472,33 @@ export const retrieveCommentRepliesConnection = (
   });
 };
 
+/**
+ * retrieveCommentAllChildCommentsConnection returns a Connection<Comment> for
+ * a given comment's child comments
+ *
+ * @param mongo database connection
+ * @param tenantID the tenant id
+ * @param storyID the id of the story the comment belongs to
+ * @param comment the comment to retrieve child comments of
+ * @param input connection configuration
+ */
+export async function retrieveCommentAllChildCommentsConnection(
```

Comment:
Reading this makes me wonder if we might want to rename it to something like `retrieveChildrenForParent` or `retrieveAllChildrenForComment`? It doesn't quite match our existing naming, but I think we might be seeing that our current naming structure is confusing when it gets agglutinized longer and longer.

Reply:
Yes, I agree that rename would be clearer and that maybe this is a case to deviate from our existing naming structure. Will update to `retrieveChildrenForParentConnection` 👍